WriteAndReadRecordsBlocksWithStrings.cpp

```diff
@@ -32,7 +32,7 @@ int main(){
 		if( p.id > 10 )
 			std::cout
 				<< p.id << ", "
-				<< UnpackString(p.name)
+				<< p.name.UnpackString()
```

Comment:
Hola, Ariel.
Al ser una función no miembro no nos metemos en "notación orientada a objetos".
Es cierto que con at de array y length the string lo estamos haciendo.
Dejame procesar esta propuesta y después te digo.

Reply:
Como dije, se puede agregar en el header una función UnpackString que tome un `String<N>` y utilice la conversión a `std::string`, de forma que sea no miembro